README.md

```diff
@@ -83,6 +83,7 @@ Symbol                              | NET35              | NET40              |
 `FEATURE_SYSTEM_CONFIGURATION`      | :white_check_mark: | :white_check_mark: | :white_check_mark: | :no_entry_sign:
 `FEATURE_TARGETEXCEPTION`           | :white_check_mark: | :white_check_mark: | :white_check_mark: | :no_entry_sign:
 `FEATURE_TEST_COM`                  | :white_check_mark: | :white_check_mark: | :white_check_mark: | :no_entry_sign:
+`FEATURE_TEST_DATASET`              | :white_check_mark: | :white_check_mark: | :white_check_mark: | :no_entry_sign:
```

Comment:
I called this `FEATURE_TEST_DATASET` (instead of just `FEATURE_DATASET`) to give a hint that `DataSet` is only used by tests, and not by the main library.